test/lib/utils/group-helper.test.js

```diff
@@ -0,0 +1,57 @@
+const { parse, sep } = require('path');
+const GroupHelper = require('../../../lib/utils/group-helper');
+
+var GroupHelperOptions, GroupHelperObj, entryPath;
+
+/**
+ * It process the entry path came from the output and checks if
+ * entry path mentioned in the group helper input is matched or not
+ * @param {string} entryPathFromOutput - entry path came from the output of the group helper
+ * @param {string} EntryFromGroupHelperInput - entry path passed as an input to the group helper
+ * @returns {boolean} - returns whether entry paths are matched or not
+ */
+const isEntryOuputMatchedInput = (entryPathFromOutput, EntryFromGroupHelperInput) => {
+    let dir = parse(entryPathFromOutput).dir;
+    let base = parse(entryPathFromOutput).base;
+    let entryOutput = dir.concat(`/${base}`).split(sep);
+    return entryOutput.join('/').includes(EntryFromGroupHelperInput);
+};
```

Comment:
I didnt place this function to test-utils because I think its quite specific to `group-helper` only. 
If you want this to be in utils, please let me know I will move it